firestore/FirestoreSwiftUIExample/Views/ContentView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ContentView.swift
+//  FirestoreSwiftUIExample
+//
+
+import SwiftUI
+import Firebase
+
+struct ContentView: View {
```

Comment:
I think it might be nice to choose a more specific name, e.g. "RestaurantListView" or "RootView".